docs/cookbook/disabling-guest-checkout.rst

```diff
@@ -0,0 +1,25 @@
+How to disable guest checkout?
+==============================
+
+Sometimes, depending on your use case, you may want to resign from the guest checkout feature provided by Sylius.
+
+In order to require users to have an account and be logged in before they can make an order in your shop,
+you have to turn on the firewalls on the ``/checkout`` urls.
+
+To achieve that simple add this path to ``access_control`` in the ``security.yml`` file.
+
+.. code-block:: yaml
+
+    # app/config/security.yml
+    security:
+        access_control:
+            - { path: "%sylius.security.shop_regex%/checkout", role: ROLE_USER }
+
+That will do the trick, now when a guest user tries to click the checkout button in the cart,
```

Comment:
`now when a guest user` looks like a starting a new sentence for me... `trick. Now, when a guest user`?